packages/notifications-library/src/notifications_library/templates/on_share_project.email.content.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+{% block title %} {% include 'on_share_project.email.subject.txt' %} {% endblock %}
+{% block content %}
+<p>Dear {{ user.first_name }},</p>
```

Comment:
be aware that sometimes the user did not feel its profile. I would always add a fallback in that case
```
<p>Dear {{ user.first_name or user.user_name or 'Valued User' }},</p>
```

NOTE: i.e. we have to add `UserData.user_name`

Reply:
This is what we have in all templates... I will change them all